examples/winrm/windows_monitor_iplatency.json

```diff
@@ -0,0 +1,41 @@
+
+{
+    "logo": "windows.svg",
+    "version": "1.0.0",
+    "name": "Windows Monitor IP Latency",
+    "description": "Monitor IP Latency from a host to another",
+    "category": "Software Integration",
+    "requirements": {
+        "credentials": true,
+        "sandbox_version": "1.12",
+        "others": [
+            "Powershell Version 5.1.19041.2364",
+            "WinRM Enabled",
+            "ping"
+        ]
+    },
+    "tags": [
+        "windows",
+        "ip address",
+        "latency",
+        "packet loss"
+    ],
+    "sample_period": 30000,
```

Comment:
This is supposed to be the recommended sample for the driver in seconds.
I doubt that 8.33 hours are a good sampling frequency. I recommend changing this to 900 or 1800 (eg 15/30 minutes)